src/Containers/DropArea/DropArea.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react';
+import { DropArea, IDropAreaProps } from './DropArea';
+
+export default {
+    title: 'Components/DropArea',
+    component: DropArea,
+    args: {},
+    argTypes: {
```

Comment:
Where is on Button Click, look at the picture I sent you, let's improve the UI

Reply:
there is no onClick, it's handled automatically by the input element ( <input {...getInputProps()}/>) where getInputProps comes from dropzone.

Reply:
and also by the div which contains the input (<div {...getRootProps()} />), where getRootProps comes again from dropzone